lisa/sut_orchestrator/azure/common.py

```diff
@@ -336,11 +315,16 @@ def _get_info(self, platform: "AzurePlatform") -> Dict[str, Any]:
 
     def _parse_info(self, raw_features: Dict[str, Any], log: Logger) -> None:
         """Parse raw image tags to AzureImageSchema"""
-        self._parse_architecture(raw_features, log)
-        self._parse_disk_controller_type(raw_features, log)
-        self._parse_hyperv_generation(raw_features, log)
-        self._parse_network_data_path(raw_features, log)
-        self._parse_security_profile(raw_features, log)
+        if self.architecture is None:
```

Comment:
For `self` and the `raw_feature`, where are they from?

Reply:
`self` is the `AzureImageSchema` parsed from the runbook (default `None`)
`raw_feature` is from quering AzCLI for image capabilities